lib/src/element_type.dart

```diff
@@ -293,11 +286,13 @@ abstract class DefinedElementType extends ElementType {
   String createLinkedReturnTypeName() => returnType.linkedName;
 
   Iterable<ElementType> _typeArguments;
+
+  /// An unmodifiable list of this element type's parameters.
   Iterable<ElementType> get typeArguments {
     _typeArguments ??= (type as ParameterizedType)
```

Comment:
This could be ` => _typeArguments ??=...`

Reply:
Good point, I've switched to an expression body. Thanks.